apps/coscrad-frontend/src/components/resources/digital-text/new-page-form.tsx

```diff
@@ -0,0 +1,51 @@
+import { isNonEmptyString } from '@coscrad/validation-constraints';
+import { Button, Stack, TextField } from '@mui/material';
+import { useState } from 'react';
+
+interface NewPageFormProps {
+    onSubmitPageIdentifier: (pageIdentifier: string) => void;
+    existingPageIdentifiers: string[];
+}
+
+export const NewPageForm = ({
+    onSubmitPageIdentifier,
+    existingPageIdentifiers,
+}: NewPageFormProps) => {
+    const [pageIdentifier, setPageIdentifier] = useState<string>('');
+
+    // remove leading and trailing white space
+    const parsedPageIdentifier = pageIdentifier.trim();
+
+    // TODO Ensure the back-end handles this with an explicit test case
+    const hasValidFormat = parsedPageIdentifier.indexOf(' ') === -1;
+
+    /**
+     * TODO Expose validation errors to the user
```

Comment:
We should add a story for this. We may want a full on epic for Digital Text UX flow.

Reply:
[Here](https://www.pivotaltracker.com/story/show/186539848) is the story.